<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="okPushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ок</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>691</width>
      <height>171</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>171</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>711</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Данные</string>
    </property>
    <addaction name="dataFromFile"/>
    <addaction name="dataFromSite"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="dataFromFile">
   <property name="text">
    <string>Данные из файла</string>
   </property>
  </action>
  <action name="dataFromSite">
   <property name="text">
    <string>Данные с сайта</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
